src/main/java/net/minecraftforge/event/entity/player/PlayerEvent.java

```diff
@@ -510,4 +511,30 @@ public PlayerChangedDimensionEvent(PlayerEntity player, RegistryKey<World> fromD
             return this.toDim;
         }
     }
+
+    /**
+     * On the server side fired when any player has their game mode changed via {@link net.minecraft.server.management.PlayerInteractionManager#setGameType(GameType)}.
```

Comment:
This "via" info makes no sense, the event docs are in need of changes and new events should not follow the existing guides.
You should describe when the event fires, not which method is responsible for it.